bigbluebutton-html5/imports/ui/components/loader/component.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```

Comment:
Component is a member of React. 
Which is why its possible to import only React, and then extend React.Component.